voicevox_engine/metas/MetasStore.py

```diff
@@ -0,0 +1,50 @@
+import json
+from pathlib import Path
+from typing import Dict
+
+from voicevox_engine.metas.Metas import SpeakerSupportPermittedSynthesisMorphing
+
+
+class MetasStore:
+    def __init__(self, engine_speakers_path: Path) -> None:
+        self._engine_speakers_path = engine_speakers_path
+        self._loaded_metas = {
```

Comment:
metasが単体のSpeakerを意図していたりDictを意図していたりの揺れがありそうでした。
この辺りはおそらくすぐバグに繋がるので、揺れをなくしつつ型情報を付けてなるべく抑えたいかもです。
```suggestion
        self._loaded_metas: Dict[str, EngineSpeaker] = {
```